detectNet.cpp

```diff
@@ -315,9 +395,18 @@ detectNet* detectNet::Create( NetworkType networkType, float threshold, uint32_t
 		return Create("third_party/jetson-inference/data/DetectNet-COCO-Chair/deploy.prototxt", "third_party/jetson-inference/data/DetectNet-COCO-Chair/snapshot_iter_89500.caffemodel", 0.0f, "third_party/jetson-inference/data/DetectNet-COCO-Chair/class_labels.txt", threshold, DETECTNET_DEFAULT_INPUT, DETECTNET_DEFAULT_COVERAGE, DETECTNET_DEFAULT_BBOX, maxBatchSize, precision, device, allowGPUFallback );
 	else if( networkType == COCO_DOG )
 		return Create("third_party/jetson-inference/data/DetectNet-COCO-Dog/deploy.prototxt", "third_party/jetson-inference/data/DetectNet-COCO-Dog/snapshot_iter_38600.caffemodel", 0.0f, "third_party/jetson-inference/data/DetectNet-COCO-Dog/class_labels.txt", threshold, DETECTNET_DEFAULT_INPUT, DETECTNET_DEFAULT_COVERAGE, DETECTNET_DEFAULT_BBOX, maxBatchSize, precision, device, allowGPUFallback );
+#if NV_TENSORRT_MAJOR > 4
+	else if( networkType == SSD_INCEPTION_V2 )
+		return Create("/home/one/nautilus/third_party/jetson-inference/data/SSD-Inception-v2/ssd_inception_v2_coco.uff", "/home/one/nautilus/third_party/jetson-inference/data/SSD-Inception-v2/ssd_coco_labels.txt", threshold, "Input", Dims3(3,300,300), "NMS", "NMS_1", maxBatchSize, precision, device, allowGPUFallback);
+	else if( networkType == SSD_MOBILENET_V1 )
+		return Create("/home/one/nautilus/third_party/jetson-inference/data/SSD-Mobilenet-v1/ssd_mobilenet_v1_coco.uff", "/home/one/nautilus/third_party/jetson-inference/data/ssd_coco_labels.txt", threshold, "Input", Dims3(3,300,300), "Postprocessor", "Postprocessor_1", maxBatchSize, precision, device, allowGPUFallback);
+	else if( networkType == SSD_MOBILENET_V2 )

```

Comment:
You can't have absolute paths, it will not scale so you need to make it third_party/jetson-inference/data/ like in tensorflow_wrapper, or you can change the API to pass in the mode via a string I think that would work better even. So you control the path in bazel.